### ISSUE TITLE
HALSendError crashes in simulation

### ISSUE THREAD

**Issue by floogulinc** [Open-RIO/ToastAPI#31]
Toast doesn't handle calls to `edu.wpi.first.wpilibj.communication.FRCNetworkCommunicationsLibrary.HALSendError()` in simulation, causing crashes anytime wpilib would call it. Right now for my robot this means that in verification (where there is no joystick) the error causes a crash. 

For example:

```
Your robot has crashed. Following is a crash log and more details.
This log has been saved to: C:\Users\floogulinc\Documents\Code\Repositories\2016-knight-fury\run\toast\crash\crash-2016-02-22_11-12-20.txt
This log will also be duplicated to USB devices, with the filename: crash-2016-02-22_11-12-20.txt
 ________     __  ____       ____  __
((       )   / / / / /_     / __ \/ /_
||  x x  |  / / / / __ \   / / / / __ \
||   ^   | / /_/ / / / /  / /_/ / / / /
\\_______/ \____/_/ /_/   \____/_/ /_/

java.lang.NoSuchMethodError: edu.wpi.first.wpilibj.communication.FRCNetworkCommunicationsLibrary.HALSendError(ZIZLjava/lang/String;Ljava/lang/String;Ljava/lang/String;Z)I
    at edu.wpi.first.wpilibj.DriverStation.reportErrorImpl(DriverStation.java:647)
    at edu.wpi.first.wpilibj.DriverStation.reportWarning(DriverStation.java:625)
    at edu.wpi.first.wpilibj.DriverStation.reportJoystickUnpluggedWarning(DriverStation.java:221)
    at edu.wpi.first.wpilibj.DriverStation.getStickAxis(DriverStation.java:244)
    at edu.wpi.first.wpilibj.Joystick.getRawAxis(Joystick.java:240)
    at com.frc2879.xboxcontroller.XboxController$Thumbstick.rawY(XboxController.java:260)
    at com.frc2879.xboxcontroller.XboxController$Thumbstick.getY(XboxController.java:339)
    at com.frc2879.knight_fury.commands.DriveTank.execute(DriveTank.java:25)
    at edu.wpi.first.wpilibj.command.Command.run(Command.java:240)
    at edu.wpi.first.wpilibj.command.Scheduler.run(Scheduler.java:222)
    at com.frc2879.knight_fury.RobotModule.tickState(RobotModule.java:77)
    at jaci.openrio.toast.core.StateTracker.tick(StateTracker.java:130)
    at jaci.openrio.toast.core.StateTracker.init(StateTracker.java:74)
    at jaci.openrio.toast.core.Toast.startCompetition_2(Toast.java:123)
    at jaci.openrio.toast.core.Toast.startCompetition(Toast.java:88)
    at jaci.openrio.toast.core.ToastBootstrap.fakeRobotBase(ToastBootstrap.java:275)
    at jaci.openrio.toast.core.ToastBootstrap.main(ToastBootstrap.java:238)
```

**Comment by LoadingPleaseWait** [comment]
This looks like a duplicate of #28.

**Comment by JaciBrunning** [comment]
Fixed in PR, now merged into development branch for Release 2.4.9
